tools/sof-kernel-log-check.sh

```diff
@@ -26,10 +26,17 @@ ignore_str="$ignore_str"'|Parity error detected'
 # https://bugzilla.kernel.org/show_bug.cgi?id=202541
 ignore_str="$ignore_str"'|xhci_hcd 0000:00:14.0: WARN Set TR Deq Ptr cmd failed due to incorrect slot or ep state'
 
-# CML Mantis occasionally throws mei_me errors 
+# CML Mantis occasionally throws Intel(R) Management Engine Interface(mei) errors 
+# https://unix.stackexchange.com/questions/109294/mei-00000016-0-init-hw-failure
 ignore_str="$ignore_str"'|mei_me 0000:00:16.0: wait hw ready failed'
 ignore_str="$ignore_str"'|mei_me 0000:00:16.0: hw_start failed ret = -62'
 
+# CML Mantis has DELL touchpad i2c error on suspend/resume
+ignore_str="$ignore_str"'|i2c_designware i2c_designware.0: controller timed out'
+ignore_str="$ignore_str"'|i2c_hid i2c-DELL0955:00: failed to change power setting'
+ignore_str="$ignore_str"'|PM: dpm_run_callback(): i2c_hid_resume+0x0/0xf0 [i2c_hid] returns -110'
+ignore_str="$ignore_str"'|PM: Device i2c-DELL0955:00 failed to resume async: error -110'
```

Comment:
@fredoh9 this change looks good. But I wonder if it would be possible to make it a bit more generic may be? Say for example, if a log reports a failed/error, can we look at the device name and ignore it if it is not in the list of devices we care about?

Reply:
> `ignore_str` is an Extended regular expression. Can you replace all the special characters (`[ ]`, `( )`, `+,`... with dots? (I'm surprised the one with brackets worked...)

@marc-hb , regarding extended regalar expression, that was good point. Special characters should be replaced in this mode. I was wondering how my test went through. It turned out that line is not required. It didn't hit out error logs hence nothing to ignore.
`
ignore_str="$ignore_str"'|PM: dpm_run_callback(): i2c_hid_resume+0x0/0xf0 [i2c_hid] returns -110'
`